src/pages/DefaultOrgSelector/DefaultOrgSelector.test.jsx

```diff
@@ -1316,12 +1318,6 @@ describe('DefaultOrgSelector', () => {
       render(<DefaultOrgSelector />, { wrapper: wrapper() })
       mocks.useIntersection.mockReturnValue({ isIntersecting: true })
 
-      const selectOrg = await screen.findByRole('button', {
```

Comment:
yea this is fine. was scratching my head for a bit, but I get it now. This interaction is just trying to mimic real usage, but the test isn't doing that anyway.

Reply:
yeah that's what I figured lol. With the mock "mocking" what the button click would have done anyway maybe there was some interaction that was tripping over itself?